lib/variable/slice.cpp

```diff
@@ -51,37 +51,39 @@ auto get_coord(const Variable &coord, const Dim dim) {
 void expect_same_unit(const Variable &coord, const Variable &value,
                       const std::string &name) {
   if (coord.unit() != value.unit()) {
-    throw except::UnitError("The unit of the " + name + " of the slice (" +
+    throw except::UnitError("The unit of the " + name + " of the slice key (" +
```

Comment:
So will this now read "The unit of the value of the slice key [...]"? Why not just "The unit of the slice key"?

Reply:
Because it distinguished between a single value, the begin, and the end values. I could change to
- 'The unit of the slice key'
- 'The unit of the slice begin'
- 'The unit of the slice end'

Is this better?

Reply:
I think so, yes.